src/style/homepage/styled.js

```diff
@@ -0,0 +1,162 @@
+import styled from "styled-components";
+import imgBackgroundMobile from "../../assets/home/background-home-mobile.jpg";
+import imgBackgroundTablet from "../../assets/home/background-home-tablet.jpg";
+import imgBackgroundDesktop from "../../assets/home/background-home-desktop.jpg";
+
+export const Container = styled.main`
+  background-image: url(${imgBackgroundMobile});
+  background-position: bottom;
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  @media (min-width: 768px) {
+    background-image: url(${imgBackgroundTablet});
+  }
+  @media (min-width: 1028px) {
+    background-image: url(${imgBackgroundDesktop});
+    flex-direction: row;
+    justify-content: space-evenly;
+    min-height: 760px;
+  }
+`;
+
+export const TextSection = styled.section`
+  width: 327px;
+  padding: 0;
+
+  @media (min-width: 768px) {
+    width: 450px;
+    height: 334px;
+    padding: 0;
+    margin: 65px 0 100px 0;
+  }
+  @media (min-width: 1028px) {
+    width: 650px;
+    margin: 0 0 0 0;
+  }
+`;
+
+export const SmallHeader = styled.p`
+  font-family: "Barlow Condensed";
+  font-size: 16px;
+  line-height: 19px;
+  text-align: center;
+  letter-spacing: 2.7px;
+  margin-top: 48px;
+  color: #d0d6f9;
+
+  @media (min-width: 768px) {
+    font-size: 20px;
+    line-height: 24px;
+    letter-spacing: 3.375px;
+  }
+
+  @media (min-width: 1028px) {
+    font-size: 28px;
+    line-height: 34px;
+    letter-spacing: 4.725px;
+  }
+`;
+
+export const BigHeader = styled.p`
+  font-family: "Bellefair";
+  font-weight: 400;
+  margin: 0;
+  color: #ffffff;
+  font-size: 80px;
+  line-height: 100px;
+  text-align: center;
+
+  @media (min-width: 768px) {
+    line-height: 150px;
+    margin: 24px 0;
+  }
+
+  @media (min-width: 1028px) {
+    font-size: 150px;
+    line-height: 172px;
+  }
+`;
+
+export const Content = styled.p`
+  color: #d0d6f9;
+  font-family: "Barlow";
+  font-size: 15px;
+  line-height: 25px;
+  text-align: center;
+
+  @media (min-width: 768px) {
+    font-size: 16px;
+    line-height: 28px;
+    text-align: center;
+  }
+
+  @media (min-width: 1028px) {
+    font-size: 18px;
+    line-height: 32px;
+    padding: 0 90px;
+    text-align: left;
+  }
+`;
+
+export const HomeButton = styled.button`
```

Comment:
try to create a common component Button that can be used everywhere if you needed